commit d4ef1b8ad8444f4077c9335b129e656d9290495e
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-26T10:45:15+01:00 (3 months ago)
    
    TRITON-1772 triton.git should reference 'joyent-repos' in repos.json
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/repos.json b/repos.json
index 8f058e8..d9fc431 100644
--- a/repos.json
+++ b/repos.json
@@ -99,6 +99,9 @@
                 "meta": true
             }
         },
+        {
+            "name": "joyent-repos"
+        },
         {
             "name": "jsstyle",
             "labels": {
